index.html

```diff
@@ -24,13 +24,17 @@
     <!--navigation bar-->
           <nav id="nav">
           <div class="nav-content">
+            <button id="menu-btn" type="button">
+              <img class="menu" id="hamburguer" src="images/menu.svg" alt="menu"/>
+            </button>
+            <img src="images/equis.svg" alt="close button" class="equis">
             <ul class="nav-items">
-              <li class="nav-item">Portfolio</li>
-              <li class="nav-item">About me</li>
-              <li class="nav-item">Contact</li>
+              <li class="nav-item"><a href="#work-section">Portfolio</a></li>
+              <li class="nav-item"><a href="#about-me">About</a></li>
+              <li class="nav-item"><a href="#contact-me">Contact</a></li>
             </ul>
           </div>
-          <img class="menu" src="images/menu.svg" alt="menu"/>
+        <!--ACA IBA LA IMAGEN DEL MENU-->
```

Comment:
- [ ] Please make sure that when the user clicks any nav link in the mobile menu, the mobile menu is closed in the process, before showing the desired section.
- [ ] Please make sure that the mobile menu takes the entirety of the page's height and width and the user can not scroll past it to the main page's content.
![image](https://user-images.githubusercontent.com/62290545/173628175-606ace4a-c295-4d7c-913e-2476f752b127.png)